subgraphs/radiant-capital/README.md

```diff
@@ -0,0 +1,171 @@
+
+
+### Deposit Balance
+> Total Deposit Balance of a Single Pool = $\sum$  value of tokens deposited in the contracts 
+
+> Deposit Balance of a Single Pool = $\sum$ Deposit Balance of single pools 
+
+
+Note: Deposit Balance must be equivalent to TVL. Users may deposit a single token among five of these supported assets: WBTC, WETH, USDT, USDC, & DAI.
+
+### Borrow Balance
+> Total Borrow Balance of the Protocol = $\sum$  value of tokens borrowed in the contracts
+
+> Borrow Balance of a Single Pool = $\sum$  Borrow Balance of single pools 
+
+Note: The maximum amount available to borrow depends on the value of assets that a user has deposited as collateral, as well as the available liquidity for the asset that a user wishes to borrow. 
+
+Important terms:
+- Loan-to-Value (LTV) - defines the maximum amount that can be borrowed against a specific collateral. 
+- For example, an LTV of 75% means that someone would be able to borrow 0.75 ETH worth of let’s say USDC, for every 1 ETH they post as collateral.
+- Naturally, since ETH is a volatile asset, the LTV will evolve with market conditions after the loan is taken.
+
+- Liquidation Threshold (LT) - is the LTV level at which the lending platform (smart contract) deems the loan to be undercollateralized and subject to liquidation.
+
+
+## Rewards
+
+### Rewards for depositors and borrowers
+
+Depositors and borrowers earn the RDNT emissions per block:
+
+>  Reward Rate of a pool = Rewards received in RDNT * RDNT Price / Deposit or borrow balance of that pool
+
+### Interest Rates
+Interest rates are determined by utilization rate. Utilization ratio represents how much of a pool has been loaned out. All borrowers pay the same rate, and all lenders receive the same rate. Additionally, the interest varies. As supply and demand change, so do the interest rates vary.
+
+### Rewards for LPs
+Liquidity providers who stake their LP tokens in Radiant earn RDNT emissions 
+Daily Rewards for Pool2 = $ Stake LP tokens value * APR
+Note: Stakers receive 20% of all RDNT emissions as an incentive for providing liquidity
+
+
+### Rewards for Vesting and Locking RDNT
+- Users who vest their RDNT earn 50% of the platform fees, generated from borrowing interest. They have the option to exit early within 28 days (four weeks) with a 50% penalty fee. The 50% penalty fees are distributed to RDNT lockers.
+Reward for vested RDNT = Platform fees 
+
+- Users who lock their RDNT earn 50% of the platform fees too, generated from borrowing interest. They also receive penalty fees from users from exit early from their vested tokens. 
+Rewards for locked RDNT = Platform fees + Penalty fees
+
+Note: 20% of all RDNT emissions are allocated to Pool2 (RDNT/WETH) rewards.
+Reference: https://docs.radiant.capital/docs/project-info/vesting-and-locking/pool2
+
+## Protocol owned liquidity
+3% of RDNT allocation is for Treasury and Liquidity.
+Reference: https://docs.radiant.capital/docs/project-info/rdnt-token
```

Comment:
To account for protocol owned liquidity, do we only count LPs owned by the treasury? or both LPs and liquid RDNT kept at the treasury address?

Reply:
Hi I think we do not count RDNT tokens as protocol owned liquidity. Treasury is not protocol owned liquidity. Will update this part to make it clear.